WordPressComKit/RequestRouter.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Alamofire
+
+public enum RequestRouter: URLRequestConvertible {
+    static let baseURLString = "https://public-api.wordpress.com/rest/v1.1/"
+    // FIXME: This needs to go somewhere better
+    public static var bearerToken = ""
```

Comment:
Yup this was a shortcut I felt bad about :smile: 
Logged: https://github.com/Automattic/WordPressComKit/issues/4